### ISSUE TITLE
link focus state is not well branded in V4

### ISSUE THREAD

**Issue by Lausselloic** [Orange-OpenSource/Orange-Boosted-Bootstrap#85]
as in boosted v3 :focus and :hover states are the same : 
```
a:hover, a:focus {
    color: #f16e00;
    text-decoration: none;
} 
```
In v4 a:focus state is only outlined but not colored, where hover state is color only.
=> Need to update v4 to match the brand and set focus state at least colored (could keep outline as it's a good way to identify focus position)

**Comment by Lausselloic** [comment]
Some discussion about that on Bootstrap : https://github.com/twbs/bootstrap/pull/26370#event-1595230360

Maybe some arguments to discuss with the Brand?

**Comment by ygatesoupe** [comment]
Fixed in commit 0243337caef07ff

Will ship in next release
